hwilib/devices/ledger.py

```diff
@@ -136,6 +137,10 @@ def __init__(self, path: str, password: str = "", expert: bool = False) -> None:
 
         self.app = btchip(self.dongle)
 
+        app_name = self.app.getAppName()
+        if app_name is not None and app_name != "Bitcoin" and app_name != "Bitcoin Test" and app_name != "app":
```

Comment:
`if app_name not in ["Bitcoin", "Bitcoin Test", "app"]`

Reply:
Done